### PR TITLE
Create Dutch translation for access-tracking.txt

### PR DESCRIPTION
**Pull request by dutchmartin**

### CODE REVIEW

File: templates/nl/access-tracking.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[data_portability>
+Stel alstublieft de persoonlijke gegevens over mij, die ik u heb verstrekt, voor mij beschikbaar in een gestructureerd, algemeen gebruikt en machine leesbaar formaat zoals vastgesteld in artikel 20, lid 1, AVG.
+]
+Mijn verzoek omvat expliciet [runs>het volgende, evenals ]alle andere services en bedrijven waarvoor u de controleur bent zoals gedefinieerd in artikel 4, lid 7, AVG[runs>: {runs_list}].
+
+Zoals bepaald in Artikel 12, lid 3 AVG, you have to provide the requested information to me without undue delay and in any event within one month of receipt of the request. According to Article 15(3) GDPR, moet u mij de gevraagde informatie verstrekken zonder onnodige vertraging en in elk geval binnen een maand na ontvangst van het verzoek. Volgens artikel 15, lid 3, AVG moet u dit verzoek zonder kosten voor mij beantwoorden.
+
+Vriendlijk verzoek ik u om mij te vertellen welke informatie (cookie nummer, apparaatkenmerk, etc.) nodig is voor u om mijn identiteit vast te stellen en hoe ik die informatie kan verkrijgen.
+[has_fields>
+Ik neem de volgende informatie op die nodig is om mij te identificeren:
```
Comment (baltpeter) [inline templates/nl/access-tracking.txt]: Isn't that the text from `access-default`? For access tracking, this should be `My contact details are as follows:`.

File: templates/nl/access-tracking.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+3. de categorieën van de betrokken persoonlijke gegevens;
+4. de ontvangers of categorieën van ontvangers aan wie de persoonlijke gegevens zijn of zullen worden bekendgemaakt;
+5. waar mogelijk, de beoogde periode waarvoor de persoonlijke gegevens worden opgeslagen, of, indien niet mogelijk, de criteria die worden gebruikt om die periode te bepalen;
+6. waar de data niet direct van de betrokkene verzameld is, alle beschikbare informatie over de bron;
+7. het bestaan van geautomatiseerde besluitvorming, inclusief profilering, als bedoeld in artikel 22, leden 1 en 4, AVG en, althans in die gevallen, zinvolle informatie over de betrokken logica, evenals het belang en de beoogde gevolgen van dergelijke verwerking voor mij.
+
+Als u mijn persoonlijke gegevens overdraagt naar een derde land of een internationale organisatie, verzoek ik om te worden geïnformeerd over de passende waarborgen overeenkomstig met artikel 46 AVG betreffende de overdracht.
+[data_portability>
+Stel alstublieft de persoonlijke gegevens over mij, die ik u heb verstrekt, voor mij beschikbaar in een gestructureerd, algemeen gebruikt en machine leesbaar formaat zoals vastgesteld in artikel 20, lid 1, AVG.
```
Comment (dutchmartin) [inline templates/nl/access-tracking.txt]:
```suggestion
Stel alstublieft de persoonlijke gegevens over mij, die ik u heb verstrekt, voor mij beschikbaar in een gestructureerd, algemeen gebruikt en door een machine leesbaar formaat zoals vastgesteld in artikel 20, lid 1, AVG.
```

File: templates/nl/access-tracking.txt (hunk 1 above)
Comment (dutchmartin) [inline templates/nl/access-tracking.txt]:
```suggestion
Mijn contactgegevens zijn:
```

Comment (dutchmartin) [reply templates/nl/access-tracking.txt]: Thank you for catching that one!

**Pull request merged**